internal/transport/http2_client.go

```diff
@@ -824,6 +825,12 @@ func (t *http2Client) NewStream(ctx context.Context, callHdr *CallHdr) (*Stream,
 			return nil, &NewStreamError{Err: err, AllowTransparentRetry: true}
 		}
 		if success {
+			// drain client transport if nextID > MaxStreamID, which is set to
+			// 75% of math.MaxUint32, which then signals gRPC to restart transport
+			// for subsequent RPCs.
```

Comment:
maybe "which signals gRPC that the connection is closed and a new one must be created for subsequent RPCs"?  "Restart transport" doesn't really imply the right things, and also this doesn't happen automatically (the LB policy would need to request it).